swagger/src/main/scala/org/http4s/rho/swagger/package.scala

```diff
@@ -97,4 +97,6 @@ package object swagger {
       t <:< typeOf[Task[_]]
   }
 
+  /** marker trait for responses returning files */
+  trait SwaggerFileResponse
```

Comment:
I make this comment having spent only a few minutes considering it, so please bear with me. 😄

Could we make this a case class with a type parameter and an `EntityEncoder` implementation that simply resolves the `EntityEcoder` for the wrapped type and forwards to it? That way we don't need to extend anything in the future and all existing `EntityEncoder`s should still work on the underlying type.

Something to the tune of:
```scala
final case class SwaggerFileResponse[T](value: T)

object SwaggerFileResponse {
  implicit def entityEncoder[T](implicit encoder: EntityEncoder[T]): EntityEncoder[SwaggerFileResponse[T]] = ???
}
```